.github/workflows/wheel_linux_ppc64le.yml

```diff
@@ -0,0 +1,64 @@
+name: Wheels
+on:
+  push:
+    branches:
+      - master
+
+env:
+  CIBW_BUILD: 'cp37-* cp38-* cp39-*'
+
+  # MacOS specific build settings
+  CIBW_BEFORE_ALL_MACOS: |
+    brew uninstall --force oclint
+    brew install gcc libomp
+
+  # Python build settings
+  CIBW_BEFORE_BUILD: |
+    pip install pybind11
+
+  # Skip PPC tests due to lack of numpy/scipy wheel support
+  CIBW_TEST_SKIP: "*-manylinux_{ppc64le}"
```

Comment:
Sorry for the questions on the previous script, I think I realise that this is supposed to be a common preamble for all of them.

Reply:
The original single wheel script had a global preamble, which I should have tidied a little on porting to the different arch files. There is a little repetition here, but until GH Actions allow the inclusion of external common files, I think the little repetition is no harm. 

When the actions interpreter is a little more mature, we should be able to refactor these into something more friendly.

Reply:
> but until GH Actions allow the inclusion of external common files

This one really breaks my heart 💔 Come on GitHub actions 🤞